src/settings-ui/Microsoft.PowerToys.Settings.UI.Library/SettingsUtils.cs

```diff
@@ -46,37 +46,45 @@ public void DeleteSettings(string powertoy = "")
+
+            // If the file needs to be modified, to save the new configurations accordingly.
+            if (deserializedSettings.UpgradeSettingsConfiguration())
+            {
+                SaveSettings(deserializedSettings.ToJsonString(), powertoy, fileName);
+            }
+
+            return deserializedSettings;
+        }
+
         /// <summary>
         /// Get a Deserialized object of the json settings string.
         /// This function creates a file in the powertoy folder if it does not exist and returns an object with default properties.
         /// </summary>
         /// <returns>Deserialized json settings object.</returns>
-        public T GetSettings<T>(string powertoy = DefaultModuleName, string fileName = DefaultFileName)
+        public T GetSettingsOrDefault<T>(string powertoy = DefaultModuleName, string fileName = DefaultFileName)
             where T : ISettingsConfig, new()
         {
-            if (SettingsExists(powertoy, fileName))
+            try
             {
-                try
-                {
-                    // Given the file already exists, to deserialize the file and read it's content.
-                    T deserializedSettings = GetFile<T>(powertoy, fileName);
-
-                    // If the file needs to be modified, to save the new configurations accordingly.
-                    if (deserializedSettings.UpgradeSettingsConfiguration())
-                    {
-                        SaveSettings(deserializedSettings.ToJsonString(), powertoy, fileName);
-                    }
-
-                    return deserializedSettings;
-                }
+                return GetSettings<T>(powertoy, fileName);
+            }
 
-                // Catch json deserialization exceptions when the file is corrupt and has an invalid json.
-                // If there are any deserialization issues like in https://github.com/microsoft/PowerToys/issues/7500, log the error and create a new settings.json file.
-                // This is different from the case where we have trailing zeros following a valid json file, which we have handled by trimming the trailing zeros.
-                catch (JsonException ex)
-                {
-                    Logger.LogError($"Exception encountered while loading {powertoy} settings.", ex);
-                }
+            // Catch json deserialization exceptions when the file is corrupt and has an invalid json.
+            // If there are any deserialization issues like in https://github.com/microsoft/PowerToys/issues/7500, log the error and create a new settings.json file.
+            // This is different from the case where we have trailing zeros following a valid json file, which we have handled by trimming the trailing zeros.
+            catch (Exception ex) when (ex is FileNotFoundException || ex is JsonException)
+            {
+                Logger.LogError($"Exception encountered while loading {powertoy} settings.", ex);
```

Comment:
Before, we didn't write anything to logs if the settings file is not found. This is expected when you start PT for the first time, so it shouldn't be an error. We could use `LogInfo` in this case.

Reply:
Agreed, let's use this opportunity to fix the logs, since this is a problem with other modules as well, where we log as errors things that are not fatal.

Reply:
Log info when the settings file is not found.